### PR TITLE
[Maintenance] Remove `bootstrap-shop` from upmerges

### PR DESCRIPTION
**Pull request by NoResponseMate**
| Q               | A
|-----------------|-----
| Branch?         | 1.14
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | blocked by #16930
| License         | MIT

### CODE REVIEW

**Review by mpysiak — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
But we should wait until the `bootstrap-shop` branch has been merged into `2.0` before we merge it 🖖🏻

**Comment by GSadee** [comment]
Thank you, Jan! :tada:

**Pull request merged**
